.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
+/.swiftpm
```

Comment:
Slight nit: this is a bit overkill -- I'd much rather have newly created shared schemes show up in git status than having them silently ignored.

(These files aren't litter.)